app/code/Magento/Newsletter/Test/Mftf/Test/CustomTestNewsletter.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="CustomTestNewsletter">
```

Comment:
Please rename test according to rules
https://devdocs.magento.com/mftf/docs/test.html#principles